tests/integration/telemetry/common/stats.go

```diff
@@ -1,628 +0,0 @@
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package common
-
-import (
-	"context"
-	"fmt"
-	"path/filepath"
-	"testing"
-	"time"
-
-	"golang.org/x/sync/errgroup"
-
-	"istio.io/istio/pkg/test"
-	"istio.io/istio/pkg/test/echo/common"
-	"istio.io/istio/pkg/test/echo/common/scheme"
-	"istio.io/istio/pkg/test/env"
-	"istio.io/istio/pkg/test/framework"
-	"istio.io/istio/pkg/test/framework/components/cluster"
-	"istio.io/istio/pkg/test/framework/components/echo"
-	"istio.io/istio/pkg/test/framework/components/echo/check"
-	cdeployment "istio.io/istio/pkg/test/framework/components/echo/common/deployment"
-	"istio.io/istio/pkg/test/framework/components/echo/common/ports"
-	"istio.io/istio/pkg/test/framework/components/echo/deployment"
-	"istio.io/istio/pkg/test/framework/components/echo/match"
-	"istio.io/istio/pkg/test/framework/components/istio"
-	"istio.io/istio/pkg/test/framework/components/istio/ingress"
-	"istio.io/istio/pkg/test/framework/components/namespace"
-	"istio.io/istio/pkg/test/framework/components/prometheus"
-	"istio.io/istio/pkg/test/framework/features"
-	"istio.io/istio/pkg/test/framework/label"
-	"istio.io/istio/pkg/test/framework/resource"
-	"istio.io/istio/pkg/test/framework/resource/config/apply"
-	"istio.io/istio/pkg/test/util/retry"
-	util "istio.io/istio/tests/integration/telemetry"
-)
-
-const (
-	DefaultBucketCount = 20
-)
-
-var (
-	apps cdeployment.SingleNamespaceView
-
-	mockProm echo.Instances
-	ist      istio.Instance
-	promInst prometheus.Instance
-	ingr     []ingress.Instance
-)
-
-var PeerAuthenticationConfig = `
-apiVersion: security.istio.io/v1beta1
-kind: PeerAuthentication
-metadata:
-  name: default
-spec:
-  mtls:
-    mode: STRICT
-`
-
-// GetIstioInstance gets Istio instance.
-func GetIstioInstance() *istio.Instance {
-	return &ist
-}
-
-// GetAppNamespace gets bookinfo instance.
-func GetAppNamespace() namespace.Instance {
-	return apps.Namespace
-}
-
-// GetPromInstance gets prometheus instance.
-func GetPromInstance() prometheus.Instance {
-	return promInst
-}
-
-// GetIstioInstance gets Istio instance.
-func GetIngressInstance() []ingress.Instance {
-	return ingr
-}
-
-func GetClientInstances() echo.Instances {
-	return apps.A
-}
-
-func GetTarget() echo.Target {
-	return apps.B
-}
-
-// TestStatsFilter includes common test logic for stats and metadataexchange filters running
-// with nullvm and wasm runtime.
-func TestStatsFilter(t *testing.T, feature features.Feature, expectedBuckets int) {
```

Comment:
Why the PR removes the file tests/integration/telemetry/common/stats.go?

Reply:
Please see the PR description in https://github.com/istio/istio/pull/47892. We consolidated the 2 test suites for EnvoyFilter and non-Envoyfilter since we only have non-Envoyfilter now.